modules/opensearch/includes/opensearch.search.inc

```diff
@@ -225,6 +225,8 @@ function opensearch_search_search(TingSearchRequest $ting_query) {
   if ($ting_query->useCollectionType(TingSearchRequest::COLLECTION_TYPE_SINGLE_OBJECT)) {
     $options['collectionType'] = 'manifestation';
   }
+  
+  $options['fullTextQuery'] = $ting_query->getFullTextQuery();
```

Comment:
Maybe a short comment explaining why this is needed?

Reply:
:+1: Thanks for the comment. The PR has been updated and the comment is outdated.